.gitignore

```diff
@@ -35,3 +35,4 @@ doc/log.txt
 .ropeproject/*
 .idea/*
 *.pyproj
+hyperspy/tests/misc/cython/test_cython_integration.c
```

Comment:
I think in the future something more dynamic and more clever should be thought up with pyx->c extension clean up.
I would also add:
`hyperspy/tests/misc/cython/test_cython_integration.c`